tasks/agent_event.rb

```diff
@@ -0,0 +1,37 @@
+#!/opt/puppetlabs/puppet/bin/ruby
```

Comment:
Add docs

Reply:
Just like other simpler tasks: What docs would be useful that aren't handled by the JSON task definition? The Ruby code is pretty basic.